### PR TITLE
Fix ClamAV database issue

### PR DESCRIPTION
**Pull request by njseeto**
### Add safebrowsing
ClamAV has removed the ability to download the safebrowsing.cvd, as a result we need to ensure SafeBrowsing is equal to 'no'.
This is outlined in this issue: mko-x/docker-clamav#103

### Copy ClamAV database files from ECR image
ClamAV have limited the download of main.cvd, daily.cvd and bytecode.cvd files, as well as limiting the use of wget, see:
https://www.mail-archive.com/clamav-users@lists.clamav.net/msg49810.html
To get around this, we have built an ECR image that holds these three files. The Dockerfile copies these files from the ECR image.

### Update Readme

Co-authored by: Brendan Butler <brendan.butler@digital.justice.gov.uk>
Co-authored by: Matt Tei <matt.tei@digital.justice.gov.uk>
Co-authored by: Tomas Destefi <tomas.destefi@digital.justice.gov.uk>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,16 @@ TCPAddr localhost # or wherever you setup this container
...
 
+ClamAV have databases that can be downloaded manually, these can be found here:
+- http://database.clamav.net/main.cvd
+- http://database.clamav.net/daily.cvd
+- http://database.clamav.net/bytecode.cvd
+
+The ClamAV moderators limited the ability to download these files via ways such as `wget` and `curl` due the enormous number of daily downloads, more details [here](https://www.mail-archive.com/clamav-users@lists.clamav.net/msg49810.html).
+
+As a result, we have built a ECR image that holds the 3 files we require, `main.cvd`, `daily.cvd` and `bytecode.cvd`.
+The [Dockerfile](Dockerfile) copies these 3 files from the image, freshclam should update these databases in the background, however, any issues with the clamav databases might be resolved by manually downloading the files and updating the ECR image.
```
Comment (brenetic) [inline README.md]:
```suggestion
The [Dockerfile](Dockerfile) copies these 3 files from the image, freshclam should update these databases in the background, however, any issues with the clamav databases might be resolved by manually downloading the files and updating the `fb-av:base` ECR image.
```

**Comment by brenetic** [comment]
great stuff. just one update please 🚀

**Review by tomas-stefano — DISMISSED** [review]
Awesome work! After the Brendan's suggestion all good 🚀

**Review by mattempty — APPROVED** [review]
🚀

**Pull request merged**